client/src/index.js

```diff
@@ -12,14 +12,65 @@ import "./index.scss";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
 import rootReducer from "./reducers";
-
+import zIndex from "@material-ui/core/styles/zIndex";
 const store = createStore(rootReducer, applyMiddleware(thunk));
 
-ReactDOM.render(
+const footerStyle = {
+  backgroundColor: "#fff",
+  fontSize: "10px",
+  boxShadow: "0 2px 6px #bababa",
+  textAlign: "center",
+  padding: "20px",
+  position: "fixed",
+  left: "0",
+  bottom: "0",
+  height: "60px",
+  width: "100%",
+  zIndex: "3",
+};
+const links = {
+  color: "silver",
+  padding: "6px",
+  textDecoration: "underline",
+};
+// A function that routes the user to the right place
+// after login
+const onRedirectCallback = (appState) => {
+  history.push(
+    appState && appState.targetUrl
+      ? appState.targetUrl
+      : window.location.pathname,
+  );
+};
+
+const Footer = () => (
+  <footer style={footerStyle}>
+    <span style={{ fontWeight: "bold" }}>
+      Copyright 2020 Fight Pandemic. All rights reserved.
+    </span>
+    <br />
+    <a href="http://www.fightpandemics.com/terms-conditions" style={links}>Terms &amp; conditions</a>|
```

Comment:
I think we will have pages for terms-conditions, privacy-policy, and cookies-policies so we'll probably use  relative paths for these links

Reply:
Yea it will be an efficient approach.

Reply:
will these pages be  static and outside of the react app?